docs/content/en/docs/packages/cluster-autoscaler/addclauto.md

```diff
@@ -120,47 +104,12 @@ eksctl anywhere apply package -f cluster-autoscaler.yaml
 
 Cluster Autoscaler will automatically be upgraded when a new bundle is activated.
```

Comment:
What is a new bundle and when does it get activated?

Reply:
Can I prevent that upgrade from happening if I don't want to change my Cluster Autoscaler version?